werkomgeving/gegeven/index.md

```diff
@@ -20,29 +20,29 @@ In bovenstaande voorbeelden wordt met [Jan] de verwijzing bedoeld naar een speci
 
 ## Een gegevenstype
 
-Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten:
+Een gegevenstype is *een typering van gelijksoortige gegevens*. Met een gegevenstype beschrijven we hoe een afzonderlijk gegeven er uit ziet, waarbij bovenstaande vier elementen terugkomen. Van een gegevenstype wil je de volgende zaken weten: 
 
-1. Hoe refereer je naar het object waarover het gegeven gaat? Welke identificerende eigenschappen gebruik je daarbij?
+1. Hoe refereer je naar het object waarover het gegeven gaat? Welke identificerende eigenschappen gebruik je daarbij? 
 2. Over welke eigenschap gaat zo'n gegeven? (dwz: relatie naar het CIM waarin de eigenschappen zijn beschreven)
-3. Welke waarden zijn mogelijk bij dit gegeven, welke beperkingen gelden hierbij? En hoeveel waarden zijn (gelijktijdig) mogelijk voor de betreffende eigenschap?
-4. Welke informatie is er over de uitspraak zelf nodig?
-5. In welke vorm vindt de vastlegging logisch gezien plaats, zijn er beperkingen aan de manier waarop deze vastlegging mag plaatsvinden?
+3. Welke waarden zijn mogelijk bij dit gegeven, welke beperkingen gelden hierbij? En hoeveel waarden zijn (gelijktijdig) mogelijk voor de betreffende eigenschap? 
+4. Welke informatie is er over de uitspraak zelf nodig? 
+5. In welke vorm vindt de vastlegging logisch gezien plaats, zijn er beperkingen aan de manier waarop deze vastlegging mag plaatsvinden? 
 6. Is een uitspraak over deze eigenschap van het object verplicht om vast te leggen?
 
-Vier verschillende soorten gegevenstypen zijn te onderscheiden:
```

Comment:
Zijn dit strikt gescheiden soorten gegevenstypes of kun je gegevenstypes hebben die onder meer dan een van de onderstaande gevallen vallen.

Reply:
Er is sprake van een hiërarchie, die je zou kunnen afleiden uit de definitie. In het begrippenkader zullen we dit visueel maken (onderdeel van het model)